content/blog/2020-04-30-gsod-ideas-2020.md

```diff
@@ -154,6 +154,21 @@ technical writer, [Jorge](https://github.com/jorgeorpinel).
 
    _Difficulty rating:_ Beginner-Medium<br/><br/>
 
+1. **SEO/ Site Analytics.** Our current website analytics are somewhat basic. We
+   will need to have a clear strategy to follow and improve our Search Engine
+   results, as well as to understand the behavior of our users, and this is a
+   priority for 2020. This should be a relatively simple project for someone
+   with proven experience in SEO or SRE!
```

Comment:
SRE stands for Site Reliability Engineer these days :)

Reply:
we can link a few tickets we have - review metas, improve metas, SEO ?

Reply:
Le's move it a bit down? it's a bit specific ...

Reply:
> SRE stands for Site Reliability Engineer these days :)

Yes, that's what I meant. Is it inaccurate to mention that role?

Reply:
yes, I think it's not very relevant here (SRE is close to DevOps, not sure why would it be relevant with SEO and meta tags)

Reply:
True. I was confused... I think I meant QA. OK, updated in 592bc548!